packages/trader/src/Modules/Trading/Containers/trade-chart.tsx

```diff
@@ -89,110 +96,85 @@ const TradeChart = observer((props: TTradeChartProps) => {
             );
     };
 
-    const barriers = main_barrier ? [main_barrier, ...extra_barriers] : extra_barriers;
+    const barriers: ChartBarrierStore[] = main_barrier ? [main_barrier, ...extra_barriers] : extra_barriers;
 
     // max ticks to display for mobile view for tick chart
     const max_ticks = granularity === 0 ? 8 : 24;
 
     if (!symbol || !active_symbols.length) return null;
 
     return (
-        <SmartChartSwitcher
-            allowTickChartTypeOnly={show_digits_stats || is_accumulator}
+        <SmartChartWithRef
+            ref={charts_ref}
```

Comment:
@ahmadtaimoor-deriv is this ref used in SmartCharts? I noticed that the ref is not used (assigned to) anywhere in SmartCharts, and `charts_ref` is always `undefined`. We can use `SmartChart` directly without wrapping it into `SmartChartWithRef`.

Reply:
done